website/data/docs-nav-data.json

```diff
@@ -355,19 +355,49 @@
       },
       {
         "title": "Installation",
-        "path": "api-gateway/consul-api-gateway-install"
+        "path": "api-gateway/install"
       },
       {
         "title": "Technical Specifications",
         "path": "api-gateway/tech-specs"
       },
       {
-        "title": "Common Errors",
-        "path": "api-gateway/common-errors"
```

Comment:
is there a reason why we removed this page? @trujillo-adam can we reinstate this page?